spaces/domains.json

```
@@ -366,6 +366,7 @@
   "vote.moonsama.com": "expectchaos.eth", 
   "vote.sunrisegaming-dao.com": "sunrisegaming.eth",
   "dao.babyspacefloki.com": "babyspacefloki.eth",
+  "vote.meowfinance.org" : "meowfinance.eth",
```

Please delete this space.
```suggestion
  "vote.meowfinance.org": "meowfinance.eth",
```